settings.gradle

```diff
@@ -1,18 +1,17 @@
 plugins {
-    id 'com.gradle.enterprise' version '3.16.2'
+    id 'com.gradle.develocity' version '3.17'
     id 'com.gradle.common-custom-user-data-gradle-plugin' version '1.13'
     id 'org.gradle.toolchains.foojay-resolver-convention' version '0.8.0'
 }
 
 def isCI = System.getenv('GITHUB_ACTIONS') != null
 
-gradleEnterprise {
+develocity {
     server = "https://ge.solutions-team.gradle.com"
     buildScan {
-        capture { taskInputFiles = true }
+        capture { fileFingerprints = true }
```

Comment:
Please remove this line since it is now enabled by default

Reply:
Done